fairscale/nn/misc/checkpoint_activations.py

```diff
@@ -3,13 +3,12 @@
 # This source code is licensed under the BSD license found in the
 # LICENSE file in the root directory of this source tree.
 
-import functools
 from typing import Any, Dict, Optional, Tuple
 
 import torch
 from torch import Tensor
 import torch.nn as nn
-import torch.utils.checkpoint as checkpoint
+import torch.utils.checkpoint as pt_checkpoint
```

Comment:
nit: it wasn't clear what "pt" was at first. I would recommend pytorch_checkpoint or torch_chkpoint if you want to shorten it.

Reply:
Similar comment for the test class.

Reply:
good idea.